.github/workflows/tests.yaml

```diff
@@ -74,7 +74,7 @@ jobs:
           go-version-file: './go.mod'
           check-latest: true
       - name: Install addlicense
-        run: go install github.com/google/addlicense@latest
+        run: go install github.com/google/addlicense@v1.0.0
```

Comment:
small nit to pin by sha here too:
`          go install github.com/google/addlicense@2fe3ee94479d08be985a84861de4e6b06a1c7208`

Reply:
done